demo

```diff
@@ -42,6 +47,19 @@ while [ -n "$1" ]; do
     shift
 done
 
+# Check if the demo-artifact has been downloaded,
+# or if there exists a newer one in storage.
+DEMO_ARTIFACT_NAME="mender-demo-artifact.mender"
+wget -q --timestamping https://dgsbl4vditpls.cloudfront.net/${DEMO_ARTIFACT_NAME}
+
+retval=$?
+if [ $retval -ne 0 ]; then
+    echo "Failed to download the demo artifact"
+    exit $retval
+fi
+
+ARTIFACT_SIZE_BYTES=$(stat -c %s ${DEMO_ARTIFACT_NAME})
```

Comment:
As we just learn in [another PR](https://github.com/mendersoftware/mender/pull/404), this will not work for Mac OS X.